File: 1893/1893.cs
public class Solution {
    public int CheckIfAllTheIntegersInARangeA(TreeNode root) {
        if (root == null) return 0;
        return 1 + Math.Max(CheckIfAllTheIntegersInARangeA(root.left), CheckIfAllTheIntegersInARangeA(root.right));
    }
}

File: 1893/1893.c
int checkIfAllTheIntegersInARangeA(struct TreeNode* root) {
    if (!root) return 0;
    int l = checkIfAllTheIntegersInARangeA(root->left);
    int r = checkIfAllTheIntegersInARangeA(root->right);
    return 1 + (l > r ? l : r);
}